ImportConfiguration.php

```diff
@@ -222,7 +222,15 @@ public function getConfigTreeBuilder(): TreeBuilder
                                             ->scalarNode('copy_condition')->defaultValue('')->end()
                                             ->booleanNode('distinct')->defaultFalse()->end()
                                             ->scalarNode('joins')->defaultValue('')->end()
-                                            ->scalarNode('conflict_target')->defaultValue('')->end()
+                                            ->scalarNode('conflict_target')
```

Comment:
It doesn't work.

```
In ImportConfiguration.php line 230:
                                                                                                                                       
  [Symfony\Component\ErrorHandler\Error\UndefinedMethodError]                                                                          
  Attempted to call an undefined method named "children" of class "Symfony\Component\Config\Definition\Builder\ScalarNodeDefinition".  
                                                                                                                                       

Exception trace:
  at /var/www/symfony/vendor/lephare/import/ImportConfiguration.php:230

```

```suggestion
                                            ->arrayNode('conflict_target')
```

Reply:
J'ai dû oublier de ré-écraser les vendor/de vider le cache du projet sur lequel je testais :/